app/controllers/api/v8/base_controller.rb

```diff
@@ -0,0 +1,44 @@
+require 'json'
+
+class Api::V7::BaseController < ApplicationController
+  clear_respond_to
+  respond_to :json
+
+  #  before_action :doorkeeper_authorize!
+  before_action :authenticate_user!
```

Comment:
You can write controller tests and make assertions on the `@current_user` instance variable. I think it's important this always works correctly :).